src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,38 @@
+package duke.task;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    String lineBreaker = "____________________________________________________________\n";
```

Comment:
Since this is a constant, perhaps you can use UPPERCASE to name it and add the keyword "final" too